utmap-server/README.html

```diff
@@ -184,7 +184,7 @@ <h1>UTMap API</h1>
     <div class="app-desc">More information: <a href="https://helloreverb.com">https://helloreverb.com</a></div>
```

Comment:
What is helloreverb.com?

Reply:
I'm actually not sure, but the html file came like that when I exported it from Swaggerhub.   Either way, it's just part of the documentation.  It doesn't affect the code.

Reply:
Not sure whether this should be left in the documentation, is this area a template for putting our own information? Either way it's not that significant so I'll resolve it